contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -10423,3 +10431,32 @@ pltsql_exec_function_cleanup(PLtsql_execstate *estate, PLtsql_function *func, Er
 	}
 	PG_END_TRY();
 }
+
+PG_FUNCTION_INFO_V1(pltsql_assign_var);
+
+Datum
+pltsql_assign_var(PG_FUNCTION_ARGS)
```

Comment:
Lets add a descriptive comment on top for the use and also about params

Reply:
Done